deployment/helm/resource-management-policies/balloons/templates/configmap_enable_nri.yaml

```diff
@@ -0,0 +1,31 @@
+{{- if .Values.nri.enable }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: patch-containerd-config
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "balloons-plugin.labels" . | nindent 4 }}
+data:
+  patch_nri.sh: |
+   #!/bin/bash
+
+   config_file="/home/config.toml" 
+   cp /etc/containerd/config.toml "$config_file"
+
+   # Check if the disable field is set to false under [plugins."io.containerd.nri.v1.nri"]
+   status=$(awk -F ' = ' '/^\[plugins\."io\.containerd\.nri\.v1\.nri"\]/{flag=1} flag && /disable/{print $2; exit}' "$config_file")
+   if [ "$status" = "false" ]; then
+      echo "NRI is already enabled..."
+   elif [ "$status" = "true" ]; then
+      # Enable the NRI plugin in containerd
+      sed '/\[plugins.\"io.containerd.nri.v1.nri/,/^$/s/disable = true/disable = false/' -i "$config_file"
+      cp -f "$config_file" /etc/containerd/config.toml
+      echo "Restarting containerd...."
+      nsenter -t 1 -m systemctl restart containerd
+      echo "NRI has now been enabled..."
+   else
+      echo "NRI plugin entry doesn't exist in the continerd config file."
+      echo "Doing nothing..."
```

Comment:
Shouldn't we add the nri plugin config in this case? The containerd config might be a stub, only containing settings that differ from the defaults.

Reply:
I was not sure. If `[plugins."io.containerd.nri.v1.nri"]` is not there, we can of course add the following defaults.
WDYT @klihub ?

```toml
    # Enable NRI support in containerd.
    disable = false
    # Allow connections from externally launched NRI plugins.
    disable_connections = false
    # plugin_config_path is the directory to search for plugin-specific configuration.
    plugin_config_path = "/etc/nri/conf.d"
    # plugin_path is the directory to search for plugins to launch on startup.
    plugin_path = "/opt/nri/plugins"
    # plugin_registration_timeout is the timeout for a plugin to register after connection.
    plugin_registration_timeout = "5s"
    # plugin_requst_timeout is the timeout for a plugin to handle an event/request.
    plugin_request_timeout = "2s"
    # socket_path is the path of the NRI socket to create for plugins to connect to.
    socket_path = "/var/run/nri/nri.sock"
```

Reply:
OR just put `disable = false` as that's the only one needed, and not pretend that we necessarily know all the nri plugin config options 

Of course this only makes sense if the containerd version that is running supports nri...